game/MiiStudio_Miidatafile.ksy

```diff
@@ -0,0 +1,142 @@
+  - id: eye_rotation
+    type: u1
+    doc: Eye rotation. Ranges from 0 to 7, down to up. Note that some eye types have a default rotation.
+  - id: eye_size
+    type: u1
+    doc: Eye size. Ranges from 0 to 7, small to large.
+  - id: eye_type
+    type: u1
+    doc: Eye type. Ranges from 0 to 59. Not ordered the same as visible in editor.
+  - id: eye_horizontal
+    type: u1
+    doc: Eye X (horizontal) distance. Ranges from 0 to 12, close to far.
+  - id: eye_vertical
+    type: u1
+    doc: Eye Y (vertical) position. Ranges from 18 to 0, low to high.
+  - id: eyebrow_stretch
+    type: u1
+    doc: Eyebrow stretch. Ranges from 0 to 6, small to large.
+  - id: eyebrow_color
+    type: u1
+    doc: Eyebrow color. Ranges from 0 to 99. Not ordered the same as visible in editor.
+  - id: eyebrow_rotation
+    type: u1
+    doc: Eyebrow rotation. Ranges from 0 to 11, down to up. Note that some eyebrow types have a default rotation.
+  - id: eyebrow_size
+    type: u1
+    doc: Eyebrow size. Ranges from 0 to 8, small to large.
+  - id: eyebrow_type
+    type: u1
+    doc: Eyebrow type. Ranges from 0 to 23. Not ordered the same as visible in editor.
+  - id: eyebrow_horizontal
+    type: u1
+    doc: Eyebrow X (horizontal) distance. Ranges from 0 to 12, close to far.
+  - id: eyebrow_vertical
+    type: u1
+    doc: Eyebrow Y (vertical) distance. Ranges from 18 to 3, low to high.
+  - id: face_color
+    type: u1
+    doc: Skin color. Ranges from 0 to 9. Not ordered the same as visible in editor.
+  - id: face_makeup
+    type: u1
+    doc: Face makeup. Ranges from 0 to 11.
+  - id: face_type
+    type: u1
+    doc: Face shape. Ranges from 0 to 11. Not ordered the same as visible in editor.
+  - id: face_wrinkles
+    type: u1
+    doc: Face wrinkles. Ranges from 0 to 11.
+  - id: favorite_color
+    type: u1
+    doc: Favorite color. Ranges from 0 to 11.
+  - id: gender
+    type: u1
+    doc: Mii gender. 0 = male, 1 = female.
+  - id: glasses_color
+    type: u1
+    doc: Glasses color. Ranges from 0 to 99. Not ordered the same as visible in editor.
+  - id: glasses_size
+    type: u1
+    doc: Glasses size. Ranges from 0 to 7, small to large.
+  - id: glasses_type
+    type: u1
+    doc: Glasses type. Ranges from 0 to 19. Not ordered the same as visible in editor.
+  - id: glasses_vertical
+    type: u1
+    doc: Glasses Y (vertical) position. Ranges from 20 to 0, low to high.
+  - id: hair_color
+    type: u1
+    doc: Hair color. Ranges from 0 to 99. Not ordered the same as visible in editor.
+  - id: hair_flip
+    type: u1
+    doc: Flip hair. 0 = no, 1 = yes.
+  - id: hair_type
+    type: u1
+    doc: Hair type. Ranges from 0 to 131. Not ordered the same as visible in editor.
+  - id: body_height
+    type: u1
+    doc: Body height. Ranges from 0 to 127, short to tall.
+  - id: mole_size
+    type: u1
+    doc: Beauty mark size. Ranges from 0 to 8, small to large.

```

Comment:
The field's `id` is `mole_size`, but the `doc` calls it a "beauty mark". For consistency it would be better to use the same term in both the `id` and the `doc`.